test/unit/rules/lint-bare-strings-test.js

```diff
@@ -51,6 +51,11 @@ generateRuleTests({
     },
 
     {
+      config: ['&#8377;'],
```

Comment:
This is invalid config I believe, please change the config to be `['₹']` instead...